pipecat-base/app.py

```diff
@@ -28,26 +27,57 @@
 )
 logger.remove()
 logger.add(sys.stderr, format=session_logger_format)
-logger.configure(extra={"session_id":"NONE"})
+logger.configure(extra={"session_id": "NONE"})
+
 
 @app.post("/bot")
-async def handle_bot_request(body: dict, x_daily_room_url: Annotated[str | None, Header()] = None, x_daily_room_token: Annotated[str | None, Header()] = None, x_daily_session_id: Annotated[str | None, Header()] = None, proxy_connection: Annotated[str | None, Header()] = None):
-    await run_default_bot(body, x_daily_room_url, x_daily_room_token, x_daily_session_id)
+async def handle_bot_request(
+    body: dict,
+    x_daily_room_url: Annotated[str | None, Header()] = None,
+    x_daily_room_token: Annotated[str | None, Header()] = None,
+    x_daily_session_id: Annotated[str | None, Header()] = None,
+    proxy_connection: Annotated[str | None, Header()] = None,
```

Comment:
This header can be removed, as it's not used anywhere.

Reply:
Fixed.